src/material-experimental/mdc-tabs/tab-label-wrapper.ts

```diff
@@ -23,12 +24,20 @@ import {MatInkBarFoundation, MatInkBarItem} from './ink-bar';
     '[attr.aria-disabled]': '!!disabled',
   }
 })
-export class MatTabLabelWrapper extends BaseMatTabLabelWrapper implements MatInkBarItem, OnDestroy {
-  _foundation: MatInkBarFoundation;
+export class MatTabLabelWrapper extends BaseMatTabLabelWrapper
+    implements MatInkBarItem, OnInit, OnDestroy {
+  _foundation = new MatInkBarFoundation(this.elementRef.nativeElement, this._document);
 
-  constructor(public elementRef: ElementRef, @Inject(DOCUMENT) _document: any) {
+  /** Whether the ink bar should fit its width to the size of the tab label content. */
+  @Input()
+  get fitInkBarToContent(): boolean { return this._foundation.fitToContent; }
+  set fitInkBarToContent(v: boolean) { this._foundation.fitToContent = coerceBooleanProperty(v); }
+
+  constructor(public elementRef: ElementRef, @Inject(DOCUMENT) private _document: any) {
```

Comment:
This should define `this._document` at the class level as type `Document`, and then set the injected `document: any` in the constructor

(we use this pattern elsewhere in the library)